server/src/main/java/org/elasticsearch/action/admin/indices/template/put/PutIndexTemplateRequest.java

```diff
@@ -79,7 +78,7 @@
 
     private Settings settings = EMPTY_SETTINGS;
 
-    private Map<String, String> mappings = new HashMap<>();
+    private String mappings;
```

Comment:
Small comment: this could be marked `@Nullable`.

Reply:
++